Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN apt-get -y update && \
         python-pip \
         xlibmesa-glu-dev \
         zlib1g-dev && \
+    rm -rf /var/lib/apt/lists/* \
```

Comment:
Just for the record I was rebuilding the image as per the minor `Dockerfile` change and it took 50m 26s.

Reply:
... and it needs ` && \` and the end so that the commands would run.

Reply:
Was that build faster or slower compared to before? For GHA it took 1h 25m  :)

Reply:
The build time that I quoted was for my laptop... so same as before, since this PR does not really change things there.